routes/api_v1_0.py

```diff
@@ -1433,9 +1439,10 @@ def after_request(response):
     header = response.headers
     # Relying on iptables to keep this safe
     header["Access-Control-Allow-Origin"] = "*"
+    header["Git-Version"] = current_app.git_version
```

Comment:
I suggest creating a prefix for any custom headers we define. `X-Onav` basically implies a namespace that any of our custom stuff will reside.
```suggestion
    header["X-ONav-Git-Version"] = current_app.git_version
```